interpreter/script/js.ml

```diff
@@ -83,7 +83,7 @@ let harness =
   "}\n" ^
   "\n" ^
   "function exports(instance) {\n" ^
-  "  return {module: instance.exports, host: {ref: hostref}};\n" ^
+  "  return {module: instance.exports, spectest: spectest};\n" ^
```

Comment:
The generated module expects an import `spectest::eq_ref`. Could you extend `spectest` above with an `eq_ref` field?

Reply:
Also, some tests expect that `hostref(1)` provide an exported wasm function, but it doesn't.

Reply:
Added missing eq_ref. I'm not sure I understand your other comment. Which test is this?

Reply:
For the test in `ref_is_null.wast:33`, `hostref(1)` is passed in as a valid value of funcref. However, afaict, it is not a valid value. When I change `hostref(1)` with `$1.exports["funcref-elem"]`, which is a valid funcref value, the test passes.

Reply:
Ah, thanks! That test actually was bogus (but the interpreter does not type-check script commands like `invoke` -- it probably should). I removed it.